data/rulei/rulei.txt

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2022 by Saugia
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+mission "Relic of the Past"
+	invisible
+	landing
+	to complete
+		never
+	npc save
+		government "Rulei"
+		personality heroic vindictive staying mute unconstrained frugal
+		system "Kanguwa"
+		ship "vyu-Ira" "Solemnity"
+
+mission "Ira Contact"
+	invisible
+	landing
+	to complete
+		never
+	on enter Kanguwa
+		fail
+		log "Factions" "Rulei" `The Rulei were likely an ancient, technologically advanced civilization from long ago.`
+			`They communicate through some sort of invasive psionic ability.`
+		conversation
+			`The second you jump into this system you begin to experience a strange, uncomfortable sensation. You feel an overwhelming sense of emptiness, but at the same time the intensity of this feeling suggests that something must be there. You barely have enough time to try and make sense of this oddity as only seconds later you start to hear ominous whispering come from behind you.`
+			choice
+				`	(Turn around.)`
```

Comment:
I'm assuming in adding other options, you mean to have different things unfold to, as if I remember correctly you don't like several choices lead to one answer. The choices are to break up the dialogue so that it's not just presented as one large text dump which is off-putting in many cases, and it's more narratively presentable if it's done more in segments. I can try to add different options, but it might be hard considering there isn't a whole lot of different things that can happen when something is whispering in your head.

Reply:
Having two options that lead to the same text is better than only having a single option.

Reply:
ALright. I wasn't sure if that was also a no-no, but if it could fit here, then I would prefer to do that to split up dialogue than not.